vidarrtest-regression.json.in

```diff
@@ -27,162 +27,142 @@
-      "fingerprintsCollector.star.runStar_peOvMMp": null,
-      "fingerprintsCollector.star.runStar_peOvNbasesMin": null,
-      "fingerprintsCollector.star.runStar_saSparsed": null,
-      "fingerprintsCollector.star.runStar_starSuffix": null,
-      "fingerprintsCollector.star.runStar_threads": null,
-      "fingerprintsCollector.star.runStar_timeout": null,
-      "fingerprintsCollector.star.runStar_transcriptomeSuffix": null,
-      "fingerprintsCollector.star.runStar_uniqMAPQ": null,
-      "fingerprintsCollector.star.indexBam_modules" : null,
-      "fingerprintsCollector.star.indexBam_timeout" : null,
-      "fingerprintsCollector.star.indexBam_jobMemory" : null,
-      "fingerprintsCollector.maxReads" : null,
-      "fingerprintsCollector.downsample.modules" : "seqtk/1.3",
-      "fingerprintsCollector.downsample.jobMemory" : null,
-      "fingerprintsCollector.downsample.timeout" : null,
-      "fingerprintsCollector.markDuplicates.modules" : "gatk/4.2.0.0",
-      "fingerprintsCollector.markDuplicates.jobMemory" : null,
-      "fingerprintsCollector.markDuplicates.timeout" : null,
-      "fingerprintsCollector.assessCoverage.modules" : "samtools/1.14",
-      "fingerprintsCollector.assessCoverage.jobMemory" : null,
-      "fingerprintsCollector.assessCoverage.timeout" : null,
-      "fingerprintsCollector.generateFingerprint.modules" : "gatk/4.2.0.0 tabix/0.2.6 hg38/p12",
-      "fingerprintsCollector.generateFingerprint.jobMemory" : null,
-      "fingerprintsCollector.generateFingerprint.timeout" : null,
-      "fingerprintsCollector.generateFingerprint.stdCC" : null,	  
-      "fingerprintsCollector.hotspots" : "/.mounts/labs/gsi/testdata/fingerprintsCollector/hotspots/dbsnp137.hg38p12.402.overlap.vcf.gz",
-      "fingerprintsCollector.runFinCreator.modules" : "python/3.6",
-      "fingerprintsCollector.runFinCreator.chroms" : null,
-      "fingerprintsCollector.runFinCreator.jobMemory" : null,
-      "fingerprintsCollector.runFinCreator.timeout" : null
+      "hotspotFingerprintsCollector.inputType": "fastq",
+      "hotspotFingerprintsCollector.outputFileNamePrefix": "TEST_0001_WG_HG38_FASTQ",
+      "hotspotFingerprintsCollector.aligner": "bwa",
+      "hotspotFingerprintsCollector.markDups" : "true",
+      "hotspotFingerprintsCollector.refFasta": "/.mounts/labs/gsi/modulator/sw/data/hg38-p12/hg38_random.fa",
+      "hotspotFingerprintsCollector.bwaMem.runBwaMem_modules": "samtools/1.9 bwa/0.7.12 hg38-bwa-index-with-alt/0.7.12",
+      "hotspotFingerprintsCollector.bwaMem.runBwaMem_bwaRef": "/.mounts/labs/gsi/modulator/sw/data/hg38-bwa-index-with-alt-0.7.12/hg38_random.fa",
+      "hotspotFingerprintsCollector.bwaMem.adapterTrimmingLog_timeout": null,
+      "hotspotFingerprintsCollector.bwaMem.adapterTrimmingLog_jobMemory": null,
+      "hotspotFingerprintsCollector.bwaMem.indexBam_timeout": null,
+      "hotspotFingerprintsCollector.bwaMem.indexBam_modules": null,
```

Comment:
see note above regarding the name,  these will all fail as they use the incorrect name (fingerprints with an s is incorrect)

Reply:
done!